addons/binding/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/YamahaReceiverBindingConstants.java

```diff
@@ -111,13 +115,21 @@
     }
 
     /**
-     * The volume min and max is the same for all supported devices.
+     * Flags indicating if a feature is supported
      */
-    public static final int VOLUME_MIN = -80;
-
-    public static final int VOLUME_MAX = 12;
-
-    public static final int VOLUME_RANGE = -VOLUME_MIN + VOLUME_MAX;
+    public enum Feature {
+        DAB,
+        TUNER,
+        SPOTIFY,
+        BLUETOOTH,
+        AIRPLAY,
+        NET_RADIO,
+        USB,
+        /**
+         * Model HTR-xxxx has a Zone_2 concept but realized as an extension to Main_Zone
+         */
+        ZONE_B
```

Comment:
The dialog level of the other PR will need an entry in this feature enum later on. Just a reminder.

Reply:
#3212